cloudformation/membership-attribute-service.yaml

```diff
@@ -236,7 +236,7 @@ Resources:
               aws s3 cp s3://gu-membership-attribute-service-dist/membership/${Stage}/membership-attribute-service/membership-attribute-service_1.0-SNAPSHOT_all.deb /tmp
               dpkg -i /tmp/membership-attribute-service_1.0-SNAPSHOT_all.deb
               mkdir -p /etc/gu
-              aws --region ${AWS::Region} s3 cp s3://gu-reader-revenue-private/membership/members-data-api/${Stage}/members-data-api.private.conf /etc/gu
```

Comment:
Is this not needed?

Reply:
It's not - it was already not on the s3 cp three lines up.